datalake/backend/endpoints/datasets/common.py

```diff
@@ -1,3 +0,0 @@
-from typing import Sequence
```

Comment:
I think it would be good to extract this change to separate PR. I like it, but it is out of scope of this PR.